data/json/monsters.json

```diff
@@ -7366,5 +7370,43 @@
     "death_function":"NORMAL",
     "description":"A baby mutant cockroach about the size of a rat.",
     "flags":["SEES", "HEARS", "SMELLS", "CHITIN", "CLIMBS"]
+    },{
+    "type" : "MONSTER",
+    "id" : "mon_zombie_survivor_elite",
+    "name": "elite survivor zombie",
+    "species":"ZOMBIE",
+    "default_faction":"zombie",
+    "symbol":"Z",
+    "color":"ltred_green",
+    "size":"MEDIUM",
+    "material":"flesh",
+    "diff":55,
+    "aggression":100,
+    "morale":100,
+    "speed":110,
+    "melee_skill":6,
+    "melee_dice":4,
+    "melee_dice_sides":8,
+    "melee_cut":0,
```

Comment:
I assume the armor multiplier is armor piercing capabilities? So how does 0.2 sound? Still makes armor work, just not as much.